config/initializers/foreman.rb

```diff
@@ -6,14 +6,14 @@
 require_dependency 'foreman/access_permissions'
 require_dependency 'foreman/plugin'
 require_dependency 'foreman/settings'
+require_dependency 'fog_extensions'
 
 # Other internal dependencies, may be autoloaded
-require_dependency 'foreman/foreman_url_renderer'
-require_dependency 'foreman/controller'
-require_dependency 'net'
-require_dependency 'foreman/provision' if SETTINGS[:unattended]
-require_dependency 'foreman'
-require_dependency 'fog_extensions'
+require 'foreman/foreman_url_renderer'
+require 'foreman/controller'
```

Comment:
this is an empty file, back from Rails 3, it is just some namespace undefined workaround, that should not be needed anymore.
It will definitelly not needed once we migrate to Zeitwerk.
Can you try if it works without it and if not, lets create a followup task after we switch to zeitwerk to remove it?